README.md

```diff
@@ -80,8 +80,54 @@ The following example instead uses "vanilla" `conda` and skips the generation of
 
 ## Outputs
 
-TBD
-
+| Output file                         |  Description                                                                                                                                                                                                               |
```

Comment:
I like this table! We should maybe drop some items that are not that useful for the users and of course complete all descriptions, but let's merge it for now so that we can all contribute